src/main/java/net/hypixel/nerdbot/command/ItemGenCommands.java

```diff
@@ -138,6 +136,82 @@ public void generateFullItem(GuildSlashEvent event,
         event.getHook().sendFiles(FileUpload.fromData(Util.toFile(merger.getImage()))).setEphemeral(hidden).queue();
     }
 
+    @JDASlashCommand(name = "itemgenparse", description = "Converts a minecraft item into a Nerd Bot item!")
+    public void parseItemDescription(GuildSlashEvent event,
```

Comment:
/infogen, located in `askForInfo()` should be updated explaining why this command exists